posthog/queries/trends/formula.py

```diff
@@ -26,11 +26,14 @@ def _run_formula_query(self, filter: Filter, team: Team):
             queries.append(sql)
             params = {**params, **entity_params}
 
-        breakdown_value = (
-            ", sub_A.breakdown_value"
-            if filter.breakdown_type == "cohort"
-            else f", {trim_quotes_expr('sub_A.breakdown_value')}"
-        )
+        breakdown_value = ""
+        if filter.breakdown_type == "cohort":
+            breakdown_values_list = ", ".join(f"sub_{letter}.breakdown_value" for letter in letters)
```

Comment:
Naming nits are best nits 😄: I'd name this `breakdown_columns`, since this is just the columns joined